runtime/compiler/trj9/env/j9method.cpp

```diff
@@ -4605,7 +4605,11 @@ TR_ResolvedJ9Method::setRecognizedMethodInfo(TR::RecognizedMethod rm)
       TR_OpaqueClassBlock *clazz = fej9()->getClassOfMethod(getPersistentIdentifier());
       J9JITConfig * jitConfig = fej9()->getJ9JITConfig();
       TR::CompilationInfo * compInfo = TR::CompilationInfo::get(jitConfig);
-      TR_PersistentClassInfo *clazzInfo = compInfo->getPersistentInfo()->getPersistentCHTable()->findClassInfo(clazz);
+      TR_PersistentClassInfo *clazzInfo = NULL;
+      if (compInfo->getPersistentInfo() && compInfo->getPersistentInfo()->getPersistentCHTable())
```

Comment:
No need for this extra check. persistentInfo always exists or will not start the JVM. It's just too essential.